gradle/versions.gradle

```diff
@@ -88,16 +88,18 @@ dependencyManagement {
     dependency 'org.apache.logging.log4j:log4j-slf4j-impl:2.13.3'
     dependency 'org.fusesource.jansi:jansi:1.8'
 
-    dependency 'org.apache.tuweni:bytes:1.2.0'
-    dependency 'org.apache.tuweni:config:1.2.0'
-    dependency 'org.apache.tuweni:crypto:1.2.0'
-    dependency 'org.apache.tuweni:devp2p:1.2.0'
-    dependency 'org.apache.tuweni:dns-discovery:1.2.0'
-    dependency 'org.apache.tuweni:io:1.2.0'
-    dependency 'org.apache.tuweni:rlp:1.2.0'
-    dependency 'org.apache.tuweni:toml:1.2.0'
-    dependency 'org.apache.tuweni:units:1.2.0'
-    dependency 'org.apache.tuweni:net:1.2.0'
+    dependencySet(group: 'org.apache.tuweni', version: '1.3.0') {
```

Comment:
What is needed from tuweni 1.3?

Reply:
I went with the latest when I updated the discovery library and also updated here. Is there a reason to stay at 1.2?

Reply:
a large scale library update like this should really be in its own PR to isolate regressions.